### DISCUSSION TITLE
Method to determine threshold for detect-content automagically

### DISCUSSION THREAD

**Discussion by cbitterfield** [Breakthrough/PySceneDetect#226] [General]
**Description of Problem & Solution**
The issue with this software is it is still very manual. I have it working. I can run the various parts. It would be more useful if it created an HTML table and file with the clips and times instead of images and a csv.

The key issue is that the scene detection and splitting need to be accurate enough not to have 15 adjustments to the thresholds and parameters. I can easily bring any media file into final cut pro (or MPEG Streamclip).  The mark and split. Handbrake, for instance, can do something similar.

I literally have 50,000 30 minutes to 2-hour videos to test against. None of the ones I have tried are even close out of the box. Since this uses OpenCV, it should be trainable to be more and more accurate. 

What propose is a means of adding learning to the OpenCV

**Media Examples:**
I can upload the source video but it is 1.3GB. 

**Proposed Implementation:**
I would propose some means for learning. Probably tied into the GUI for the product. 

**Alternative Solutions:**

**Comment by Breakthrough** [comment]
I believe a statistical method should be proposed to address this issue - in particular, some kind of peak detection, however this is still considered an experimental topic currently.
